lib/r10k/action/deploy/display.rb

```diff
@@ -52,7 +52,8 @@ def source_info(source, argv=[])
           source_info[:prefix] = source.prefix if source.prefix
           source_info[:remote] = source.remote if source.respond_to?(:remote)
 
-          env_list = source.environments.select { |env| argv.empty? || argv.include?(env.name) }
+          all_envs = requested_environments.empty?
```

Comment:
(nit) maybe call this `select_all_envs` or something? I want it to be clear that this is a boolean and not a collection of environments.

Reply:
yeah, this is one place where I'd love to make a local variable with a `?`.

Reply:
updated

Reply:
ugh same, it bums me out every time I remember that you can't do that in Ruby